test/parallel/test-cli-bad-options.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
-require('../common');
+const common = require('../common');
+common.skipIfInspectorDisabled();
```

Comment:
Nit: Instead of skipping the test, can we use it to guard just the arguments that require inspector? We should still be able to run this on `--eval`, no?

Reply:
I think so, yes. Done! :)